lambdas/data-migration2/tests/test-index.js

```diff
@@ -0,0 +1,255 @@
+const test = require('ava');
```

Comment:
This looks good, similar considerations from the previous unit (except I just noted we're using the factory here in one example)